spec/requests/dois_spec.rb

```diff
@@ -186,6 +186,8 @@
 
```

Comment:
Integration Tests  (request/doi_spec.rb) need to be adjusted to accommodate the changes.

Reply:
Please see the tests in the pull request.

Reply:
I mentioned because Travis is failing on `rspec ./spec/requests/dois_spec.rb:181 # dois GET /dois/:id when the record exists returns the Doi`